lib/cucumber/html_builder.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+require 'builder'
+
+module Cucumber
+  class HtmlBuilder < Builder::XmlMarkup
```

Comment:
I debated that with myself and don't feel that strongly one way or the other so I'll go ahead and change it.